README.md

```diff
@@ -16,6 +16,39 @@
 You'll need an API key. Follow Render's [docs](https://render.com/docs/api#getting-started) for creating one.
 Then set the API key as a secret with `pulumi config set --secret render:apiKey`.
 
+### Importing Existing Resources
```

Comment:
@VladimirCreator let me know if you find this useful.

Reply:
@praneetloke try to import registrycredentials. I tried and got 404 for some reason. Want to make sure that I’m not doing something incorrect.

I used the following command:
```
 pulumi import render:registrycredentials:RegistryCredential githubregistrycredential /registrycredentials/{yourCredentialsId}
```

Reply:
@VladimirCreator That import command looks right to me. Can you please run that with the flags `-v 3 --logtostderr --logflow` and open a new issue for it? (Please be sure to remove confidential details from it.)

Reply:
@VladimirCreator I found an issue with the import process. I am actively looking into it.

Reply:
Alright, so the first issue is that the doc for importing is incorrect 😆 I have corrected that now. But there is still an issue with importing the resource that I am looking into. It'll require a fix in `pulumi-provider-framework`.

Reply:
@VladimirCreator `v0.3.10` has the fix for the import. It was a good catch! You helped me find an edge case. However, I think it's useless to import Render's `RegistryCredential` resource because their API does not return the `authToken` on read. They also don't return the `ownerId` on read which their API requires as input to create the resource. In their UI, they supply the `ownerId` automatically as the currently-logged in user, so we don't see it. What this means is that during import you'll have to either add `ownerId` and `authToken` to `ignoreChanges` resource option or you'll have to use blank values for those two properties. There is just no way to read those properties from their API today once they are created. So you are better off just re-creating those credentials through Pulumi. I am assuming that services just take the ID of the registry credential rather than the raw secret value and Render pulls the credentials behind-the-scenes? (I haven't yet used registry credentials myself for anything.)

Reply:
@praneetloke I followed the updated docs and I tried to import web service using command:

`pulumi import … resname /myWebService’sId`

but keep getting 404.

are you able to import a web service?

Reply:
@VladimirCreator I have imported a static site and a custom domain before but not a web service. Can you share logs by running the import with `-v 3 --logtostderr --logflow`? Please feel free to open a new issue in this repo and I'll investigate asap.

Reply:
@praneetloke The problem was that I added leading /. `pulumi import render:service:WebService myres /id` does not work but `pulumi import render:service:WebService myres id` does. Will you update the docs cause the leading / is also there as well

Reply:
Oh interesting. There must be a bug in the code that maps the ID segments because the prefix is trimmed. I'll check it out and fix the bug or the docs. Thank you for reporting it to me!

Reply:
I've fixed the issue where it seemed that having a leading `/` in the import ID was causing a problem. https://github.com/cloudy-sky-software/pulumi-provider-framework/pull/325.